src/scheduler/connectivity.rs

```diff
@@ -73,33 +73,33 @@ impl DetailedConnectivity {
         }
     }
 
-    fn to_string_imap(&self, _context: &Context) -> String {
+    async fn to_string_imap(&self, context: &Context) -> String {
         match self {
-            DetailedConnectivity::Error(e) => format!("Error: {}", e),
+            DetailedConnectivity::Error(e) => stock_str::error(context, e).await,
             DetailedConnectivity::Uninitialized => "Not started".to_string(),
-            DetailedConnectivity::Connecting => "Connecting…".to_string(),
-            DetailedConnectivity::Working => "Getting new messages…".to_string(),
+            DetailedConnectivity::Connecting => stock_str::connecting(context).await,
+            DetailedConnectivity::Working => stock_str::updating(context).await,
             DetailedConnectivity::InterruptingIdle | DetailedConnectivity::Connected => {
-                "Connected".to_string()
+                stock_str::connected(context).await
             }
             DetailedConnectivity::NotConfigured => "Not configured".to_string(),
         }
     }
 
-    fn to_string_smtp(&self, _context: &Context) -> String {
+    async fn to_string_smtp(&self, context: &Context) -> String {
         match self {
-            DetailedConnectivity::Error(e) => format!("Error: {}", e),
+            DetailedConnectivity::Error(e) => stock_str::error(context, e).await,
             DetailedConnectivity::Uninitialized => {
-                "(You did not try to send a message recently)".to_string()
+                "You did not try to send a message recently.".to_string()
```

Comment:
We don't want to translate this?

Reply:
i have never seen this string in normal program flow, therefore i thought it is a minor thing, same as the error strings (which are also not translated).

Reply:
It's shown when DC was just started e.g. after a force stop or a reboot, and no message was sent yet so that the SMTP thread is not running yet.

Reply:
hm, it tried on ios/desktop/android - it tells always me that "Your last message was sent successfully.", no matter of stop or reboot. 

maybe there is a bug somewhere?